thirdeye/thirdeye-frontend/app/pods/rootcause/template.hbs

```diff
@@ -152,140 +152,122 @@
       </div>
 
       {{#if context.urns.size}}
-        <div class="card-container__header rootcause-tabs">
-          <div class="card-container__subnav">
-            <a class="thirdeye-link thirdeye-link--nav {{if (eq activeTab "metrics") "thirdeye-link--active "}}" {{action (mut activeTab)
-              "metrics"}}>
-              Metrics
-            </a>
-          </div>
-          <div class="card-container__subnav">
-            <a class="thirdeye-link thirdeye-link--nav {{if (eq activeTab "dimensions") "thirdeye-link--active"}}" {{action (mut activeTab) "dimensions"}}>
-              Dimensions
-            </a>
-          </div>
-          <div class="card-container__subnav">
-            <a class="thirdeye-link thirdeye-link--nav {{if (eq activeTab "events") "thirdeye-link--active"}}" {{action (mut activeTab) "events"}}>
-              Events
-            </a>
-          </div>
-          <div class="card-container__subnav">
-            <a class="thirdeye-link thirdeye-link--nav {{if (eq activeTab "trend") "thirdeye-link--active"}}" {{action (mut activeTab) "trend"}}>
-              Trend
-            </a>
-          </div>
-        </div>
+        {{#shared/common-tabs selection=activeTab activeTab=activeTab as |tabs|}}
```

Comment:
At first look, it seems the tabs component set will be able to handle "sub-tabs" in any given `tabpanel`. Any thoughts on that?